vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,457 @@
+
+You can visualise the score as a colour gradient using the `r3dmol` package in R. For that you need to first create a gradient using the `colorRampPalette()` function, which creates a function capable of creating a colour gradient over the specified colours. Instead of providing only three colours to this function you can provide additional colours to make a gradient with multiple substeps. It is important to always use 101 colours to get the full range for the structure. The colour gradient needs to be concatenated using `paste0()` and provided to the `colorfunc` argument within the shown javascript function context. 
+
+```{r score_3d_structure_mapping, eval = test_protti, echo=TRUE, warning=FALSE}
+# create a color gradient with 101 colors
+color_gradient <- paste0('"', 
+                         paste(colorRampPalette(c("white", "#90EE90", "#FF7276"))(101), 
+                               collapse = '", "'), 
+                         '"')
+
+# create structure
+r3dmol() %>%
+  m_add_model(data = paste0(tempdir(), "/2HWG_P08839.pdb"), format = "pdb") %>%
+  m_set_style(style = m_style_cartoon(
+    colorfunc = paste0("
+        function(atom) {
+          const color = [", color_gradient,"]
+          return color[Math.round(atom.b)]
+        }")
+  )) %>%
+  m_zoom_to()
+```
+
+Applying the score to both structures definitely refines the region that is most robustly affected by the treatment. For ptsI it seems that a region of amino acids directly in the active center are consistently affected. In addition we see a small region below, which was previously not seen due to its low fold changes that did not meet the cutoff. The score tells us that this region is indeed consistently affected and that there is likely a structural change occurring even though the fold change of peptides is rather low (based on looking at the tables). The region in the binding interface previously indicated with a black arrow is now not highlighted anymore, indicating that this was likely a false positive hit and that other peptides in the same region do not exhibit a strong and significant response.
+
+```{r peptide_map_2hwg_score, eval = test_protti, echo = FALSE, fig.align = "center", fig.cap = "**ptsI (2HWG) with mapped amino acid scores.**"}
+knitr::include_graphics("figures/peptide_map_2hwg_score.png")
+```
+
+For ptg it becomes more clear that the most robustly changing region is indeed very close to the binding site of 3PG, which the protein was treated with. In this case the score helped to refine the binding site of the compound a lot since previously basically the whole protein was covered with significantly changing peptides.
```

Comment:
```suggestion
For ptg we see that the most robustly changing region is very close to the binding site of 3PG, which the protein was treated with. In this case, the score helped to refine the binding site of the compound since previously almost the whole protein was covered with significantly changing peptides.
```